.ruby-version

```diff
@@ -1 +1 @@
-1.9.3-p392
+2.6.5p114
```

Comment:
I'm not sure if this file is necessary, but if we're going to have this file, we should probably use [the version currently in use on GH Pages](https://pages.github.com/versions/).